vars/runPipeline.groovy

```diff
@@ -344,13 +351,16 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
         "dlang/dub",
         "sociomantic-tsunami/ocean",
         "higgsjs/Higgs",
-        "BlackEdder/ggplotd",
-        "atilaneves/unit-threaded",
-        "d-gamedev-team/gfm",
-        "rejectedsoftware/diet-ng",
-        "dlang-community/D-YAML",
-        "CyberShadow/ae",
-        "Hackerpilot/libdparse",
+        "BlackEdder/ggplotd", // 1m56s
+        "dlang-community/D-Scanner", // 1m40s
+        "rejectedsoftware/diet-ng", // 56s
+        "atilaneves/unit-threaded", //36s
+        "d-gamedev-team/gfm", // 28s
+        "dlang-community/DCD", // 23s
+        "CyberShadow/ae", // 22s
+        "dlang-community/D-YAML", // 15s
+        "dlang-community/libdparse", // 13s
+        "dlang-community/dfmt", // 11s
```

Comment:
We should prolly start to write the times to a persisted json file and read this dynamically.
https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-unarchive-code-copy-archived-artifacts-into-the-workspace